configuration/parse-section.md

```diff
@@ -61,18 +61,18 @@ Here's the list of built-in parser plugins:
 
 The default value of the following parameters will be overridden by the individual parser plugins:
 
+{% hint style='info' %}
+NOTE: parse parameters will not work with `@type sample` input plugin.
+{% endhint %}
```

Comment:
Thanks for this fix!

Could you tell me why we need this note?

> NOTE: parse parameters will not work with `@type sample` input plugin.

Reply:
I don't understand this discussion in detail, but my first impression is that this is an issue of Input plugin and should be described in https://docs.fluentd.org/input/sample.

I'm sorry, but if there is something I don't understand, please let me know.

Reply:
Yes. not confused for that case.

Reply:
Fixed it.